app/world-6/farming/content.tsx

```diff
@@ -76,6 +77,8 @@ function Farming() {
         return false;
     }, [theData, farming])
 
+    const tabsToShow = farming.canLevelLandRank ? ["Plots", "Market Upgrades", "Crop Depot", "Land Rank"] : ["Plots", "Market Upgrades", "Crop Depot"];
```

Comment:
This  is fine for now but in the future this might become harder to maintain "base" tabs and "unlockable" tabs, so we can explore something like this

```suggestion
    let tabsToShow = ["Plots", "Market Upgrades", "Crop Depot"];
    if (farming.canLevelLandRank) {
        tabsToShow.push("Land Rank");
    }
```